io/post_regional.F90

```diff
@@ -160,11 +160,13 @@ subroutine post_run_regional(wrt_int_state,mypei,mpicomp,lead_write,      &
             if(mype==0) print *,'af read_xml at fh00,name=',trim(filenameflat)
           else if(ifhr > 0) then
             filenameflat = 'postxconfig-NT.txt'
-            if(size(paramset)>0) then
+            if(associated(paramset) .and. size(paramset)>0) then
```

Comment:
Same here

Reply:
How? If `associated(paramset)` (means it's a pointer) is false, how can the size be 1?

Reply:
> That is what I got when I tried with gnu compiler. The issue happened when it tried to get the size of paramset(1)%param.
> […](#)

I think it would be best to make the same changes as in lines 161-162, 165-166.